integration-tests/conftest.py

```diff
@@ -24,24 +28,103 @@ def __init__(self, rhsm_register_server_proxy, *args, **kwargs):
         self._private_bus_address = None
 
     def __enter__(self):
-        logger.debug("subscription: starting RHSM private DBus session")
         locale = os.environ.get("LANG", "")
         self._private_bus_address = self._rhsm_register_server_proxy.Start(locale)
-        logger.debug("subscription: RHSM private DBus session has been started")
         return self
 
     def __exit__(self, _exc_type, _exc_value, _exc_traceback):
-        logger.debug("subscription: shutting down the RHSM private DBus session")
         self.disconnect()
         locale = os.environ.get("LANG", "")
         self._rhsm_register_server_proxy.Stop(locale)
-        logger.debug("subscription: RHSM private DBus session has been shutdown")
 
     def _get_connection(self):
         """Get a connection to RHSM private DBus session."""
         # the RHSM private bus address is potentially sensitive
         # so we will not log it
-        logger.info("Connecting to the RHSM private DBus session.")
         return self._provider.get_addressed_bus_connection(
             bus_address=self._private_bus_address, flags=Gio.DBusConnectionFlags.AUTHENTICATION_CLIENT
         )
+
+
+"""
+A class used by a fixture dbus_current_signals.
+A class instance remembers a datetime of a test's start.
+I will provide signals (provided by 'dbus_signals' fixture) that are newer
+than a start of the test.
+"""
+
+
+class DBusSignals:
+    def __init__(self, since, signals):
+        self.since = since
+        self.signals = signals
+
+    """
+    A method for a test to read all signals captured by dbus monitor.
+    The method provides all signals that are emitted after the test begins.
+    """
```

Comment:
The same comment for this doc-string. The doc-string probably belongs to following method `read()`. It should look like this:

```python
def read(self):
   """
   A method for reading all signals captured by D-Bus monitor.
   The method returns all supported signals that are emitted after the test begins.
   """
```